### PR TITLE
Update timestamp format in the scanresults endpoint

### PR DESCRIPTION
**Pull request by rtuin**
The timestamp format was invalid.

### CODE REVIEW

File: docs/endpoints/scan-results.md [hunk 1]
```diff
@@ -27,15 +27,15 @@ The body must contain all collected data from the client's device.
       "UUID": "44c0ffee-988a-49dc-0bad-a55c0de2d1e4",
       "major": 16,
       "minor": 0,
-      "timestamp": 1406804464.128,
+      "timestamp": 1406804464,
       "timezone": "UTC+01:00",
```
Comment (frontconnect) [inline docs/endpoints/scan-results.md:31]: What generated this timestamp?

Comment (jvandijk) [inline docs/endpoints/scan-results.md:31]: Your phone! :)

Comment (jvandijk) [inline docs/endpoints/scan-results.md:31]: And it's correct... it should not be the microseconds version! +1 from me

Comment (frontconnect) [inline docs/endpoints/scan-results.md:31]: Guilty...

**Pull request merged**